packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -20,6 +20,7 @@ import ODataStore from 'data/odata/store';
 import TagBox from 'ui/tag_box';
 import { normalizeKeyName } from 'events/utils/index';
 import { getWidth, getHeight } from 'core/utils/size';
+import guid from 'core/guid';
```

Comment:
it's a class as far as I remember. Let's use code style consistent with other places
```suggestion
import Guid from 'core/guid';
```